include/swift/Reflection/TypeRefBuilder.h

```diff
@@ -372,6 +372,7 @@ class TypeRefBuilder {
                     unsigned ordinal) {
     // TODO: Produce a type ref for the opaque type if the underlying type isn't
     // available.
+    opaqueDescriptor->dump();
```

Comment:
@jckarter Is this intentional? Looks like it causes linker failures in some configurations.

Reply:
Oops, no, it can be removed. I'll take care of it.